test/utils/common.js

```diff
@@ -0,0 +1,82 @@
+import { expect, $ } from "@wdio/globals";
+import * as selectors from "./selectors";
+
+export function getSignInUrl(type) {
+  const baseUrls = {
+    apply: "http://localhost:3000/apply/endemics/dev-sign-in",
+    claim: "http://localhost:3004/claim/endemics/dev-sign-in",
+  };
+
+  const dockerUrls = {
+    apply: "http://ffc-ahwr-farmer-apply:3000/apply/endemics/dev-sign-in",
+    claim: "http://ffc-ahwr-farmer-claim:3000/claim/endemics/dev-sign-in",
+  };
+
+  return process.env.DOCKER_MODE === "true" ? dockerUrls[type] : baseUrls[type];
+}
+
+export async function fillAndSubmitSBI(sbi) {
+  await $("#sbi").setValue(sbi);
+  await clickSubmitButton();
+}
+
+export async function submitApplicationSteps() {
+  for (let i = 0; i < 3; i++) {
+    await clickSubmitButton();
+  }
+}
+
+export async function enterVisitDateAndContinue() {
+  const today = new Date();
+  const day = today.getDate().toString().padStart(2, "0");
+  const month = (today.getMonth() + 1).toString().padStart(2, "0");
+  const year = today.getFullYear().toString();
+
+  await $(selectors.visitDateDay).setValue(day);
+  await $(selectors.visitDateMonth).setValue(month);
+  await $(selectors.visitDateYear).setValue(year);
+  await clickContinue();
+}
+
+export async function whenTestingWasCarriedOutAndContinue(value) {
```

Comment:
probably want a better name for this function, they should have names that describe what they do. This one enters when the testing happened, and clicks continue, so it should have 'enter' on the front.